README.md

```diff
@@ -66,7 +66,6 @@ Full list of options in `config.json`:
 | consumer_timeout_ms                 | Integer |            | (Default: 10000) KafkaConsumer setting. Number of milliseconds to block during message iteration before raising StopIteration            |
 | session_timeout_ms                  | Integer |            | (Default: 30000) KafkaConsumer setting. The timeout used to detect failures when using Kafka’s group management facilities. |                                      |
 | heartbeat_interval_ms               | Integer |            | (Default: 10000) KafkaConsumer setting. The expected time in milliseconds between heartbeats to the consumer coordinator when using Kafka’s group management facilities. |
-| max_poll_records                    | Integer |            | (Default: 500) KafkaConsumer setting. The maximum number of records returned in a single call to poll(). |
```

Comment:
Why delete this? is it being deprecated?
There is a tap in prod that is using it.

Reply:
This option is not supported by the by the underlying librdkafka C library. The full range of configuration properties please consult librdkafka’s documentation: https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md

More info in the [confluent-kafka documentation](https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#pythonclient-configuration)

Reply:
Okay.

Can you remove the `DEFAULT_MAX_POLL_RECORDS` constant then?